pkg/project/registry/proxy/proxy_test.go

```diff
@@ -70,24 +69,10 @@ func TestCreateProjectBadObject(t *testing.T) {
 	}
 }
 
-func TestCreateProjectMissingID(t *testing.T) {
```

Comment:
Why is this test now invalid?  I don't see a replacement in another package in this pull.